features/mcp/deploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source $(dirname "$0")/../hack/common.sh
+
+oc apply -f ${MCP_DIR}/00-mcp-worker-rt.yaml
```

Comment:
if we want to tie this to func tests, we must ensure that deploy also waits for the deployment to be ready.
Once we have that, we'll be able to run ` export FEATURES="feat1 feat2" && make deploy && make functests` (see https://github.com/openshift-kni/baremetal-deploy/pull/66 )

Reply:
Never mind, I thought it was a machine config. Ignore my previous comment